src/components/Input/components/Switch/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { ReactElement } from 'react';
+import clsx from 'clsx';
+import InputBase from '../InputBase';
+import { Props } from './types';
+
+
+function Switch({ className, ...otherProps }: Props): ReactElement {
+    const switchClasses = clsx(
+        'brew-Input--Switch',
+        className,
+    );
+    const customControl = (
+        <div className="brew-Input__customControl">
```

Comment:
This class and the ones below should start with `brew-Switch__` and I would avoid using any reference to `customControl` as that is a reference to the inner working of the component. 

`brew-Switch__control`, `brew-Switch__track`, and `brew-Switch__knob` should suffice.